packages/storybook/stories/inbox/full-page-inbox.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import ReactMarkdown from "react-markdown";
+
+import { CourierProvider } from "@trycourier/react-provider";
+
+import fullPageInboxHooksString from "!raw-loader!./full-page-inbox-hooks.tsx";
+import { FullPageInboxHooks } from "./full-page-inbox-hooks";
+
+export default {
+  title: "Full Page Inbox",
+};
+
+export const Hooks = () => {
+  return (
+    <>
+      <ReactMarkdown>{"TODO"}</ReactMarkdown>
+      <div
+        style={{
+          display: "flex",
+          alignItems: "top",
+          justifyContent: "space-between",
+        }}
+      >
+        <div>
+          <ReactMarkdown>{`## Example`}</ReactMarkdown>
+          <ReactMarkdown>{`\`\`\`javascript\n${fullPageInboxHooksString}\n\`\`\``}</ReactMarkdown>
+        </div>
+        <CourierProvider
+          wsOptions={{
+            url: "wss://icnrz8ttcf.execute-api.us-east-1.amazonaws.com/staging",
```

Comment:
not sure if this matters i left staging on here.  it was how i tested the CPP changes